sdk-cpp/tests/CMakeLists.txt

```diff
@@ -3,14 +3,18 @@
 find_package(Boost COMPONENTS filesystem program_options REQUIRED)
 find_package(GTest REQUIRED)
 
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
```

Comment:
Use DO_PLATFORM_ID and DO_PLATFORM_ID_LINUX. In future, if there is a need to differentiate the platform for agent and plugin as well, then we can move the definition of DO_PLATFORM_ID* to the root cmakelists.

Reply:
Sounds good

---
In reply to: [605099852](https://github.com/microsoft/do-client/pull/18#discussion_r605099852) [](ancestors = 605099852)